charts/local-ai/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: 1.40
 description: A Helm chart for deploying LocalAI to a Kubernetes cluster
 name: local-ai
 type: application
-version: 3.3.1
+version: 3.4.0
```

Comment:
These changes are potentially breaking. I'm not sure what the right version would be.

Reply:
3.400.0 🤣 jk ... maybe 4.0 then?

Reply:
The worst thing that can happen is that the `helm upgrade` command fails because the `accessMode` should now be `accessModes` and be an array. The user can simply change the values and retry so it doesn't really "break" something, just needs the user to make some changes to the values.yaml used.

If someone was really relying on some default models being there, they should just re-implement the init container and pass it through the values.yaml but I doubt anybody would rely on a pre-selected list of models being there.